src/common/utils/format.test.js

```diff
@@ -10,7 +16,7 @@ describe('format', () => {
       expect(formatDimension(1, 'fi')).toMatch('m');
     });
 
-    it('should return "-" if the value is undefined/null', () => {
+    it('should return null if the value is undefined/null', () => {
```

Comment:
Would a dash still be better than showing null to the end user?

Reply:
True, fixed